File: Iniciante/2152.py
testes = int(input())

for i in range(testes):
  h, m, o = map(int, input().split())

  hora = str(h).zfill(2)
  minuto = str(m).zfill(2)
  
  if o == 0:
    print("{}:{} - A porta fechou!".format(hora, minuto))
  else:
    print("{}:{} - A porta abriu!".format(hora, minuto))

File: Iniciante/2152.js
let testes = Number(lines.shift());

for(let i = 0; i < testes; i++) {
  let [h, m, o] = lines.shift().split(' ').map(Number);

  let hora = h.toString().padStart(2, '0');
  let minuto = m.toString().padStart(2, '0');
  
  if (o == 0) {
    console.log(`${hora}:${minuto} - A porta fechou!`);
  } else {
    console.log(`${hora}:${minuto} - A porta abriu!`);
  }
}
